framework/src/variables/MooseVariableData.C

```diff
@@ -12,7 +12,7 @@
 #include "MooseError.h"
 #include "DisplacedSystem.h"
 #include "TimeIntegrator.h"
-#include "MooseVariableFE.h"
+#include "MooseVariableFV.h"
```

Comment:
This doesn't look right. Looks like we're getting lucky with an include from `Assembly.h`. I don't think we need the `MooseVariableFV` header right?

Reply:
fixed.